### PR TITLE
Create SkipListTest.java

### PR DESCRIPTION
**Pull request by yatharthshahjpr**
Skip List code in Java! :)

### CODE REVIEW

**Comment by yatharthshahjpr** [comment]
@AdiChat , Please review my PR! :)

File: code/data_structures/skip_list/SkipListTest.java [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    public static void main(String[] args)
+    {            
+        Scanner scan = new Scanner(System.in);
+        /* Creating object of SkipList */
+        SkipList sl = new SkipList(100000000); 
+        System.out.println("SkipList List Test\n");          
+        char ch;
+        /*  Perform list operations  */
+        do
+        {
```
Comment (AdiChat) [inline code/data_structures/skip_list/SkipListTest.java]: Instead of user input, use input defined in code to demonstrate the working of the code 👍

**Comment by AdiChat** [comment]
Kindly rectify to get this merged 👍

**Comment by yatharthshahjpr** [comment]
Actually the entry will be based upon the user and therefore....according to his/her wish the requirements will be full filled ...........
BUt I will Still Change the code if that's the issue! :)

**Comment by yatharthshahjpr** [comment]
U can Please review the Pr the code has been changes according to the requirements ! 👍

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Comment by yatharthshahjpr** [comment]
Thanks !  :)

**Pull request merged**